table/block_fetcher.h

```diff
@@ -72,6 +74,7 @@ class BlockFetcher {
   const ImmutableCFOptions& ioptions_;
   bool do_uncompress_;
   bool maybe_compressed_;
+  BlockType block_type_;
```

Comment:
Is `block_type_` mutable?

Reply:
No, it does not change. We can mark it const (similarly to every other member variable that stores an input parameter I guess).